OpenRA.Mods.Common/Activities/Air/ReturnToBase.cs

```diff
@@ -18,6 +19,8 @@ namespace OpenRA.Mods.Common.Activities
 {
 	public class ReturnToBase : Activity
 	{
+		const int TooCloseDestinationThreshold = 5000;
```

Comment:
I think that this constant is to resolve this class specific problem. It do not affect other classes or is the main indicator that the destination is too close. On my opinion we can send it to the Info class when we know a real case for the parametrization of the number.